templates/object_preview.html

```diff
@@ -1,10 +1,10 @@
 {% load thumbnail %}
 
-<a class="artist-details name" href="{{ object.url }}">
-    {% thumbnail img "250x250" crop="center" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ object.name }}" />
-    {% endthumbnail %}
-    <div class="artist-details artist-name">
-        {{ object.name }}
-    </div>
-</a>
+<div data-content="{{ object.name }}" class="name">
```

Comment:
I guess from our conversation offline I was seeing something different than you were. I'll investigate this a bit, but hopefully it won't matter on production.